updater/lib/dependabot/dependency_snapshot.rb

```diff
@@ -53,6 +53,19 @@ def job_dependencies
       end
     end
 
+    def job_group_name
+      job.dependency_group_to_refresh
+    end
```

Comment:
My thinking is that in future we might have a `dependency_group_to_create` key to allow us to just build a single group if we want to change the workload profile of group/version updates.

For now it might make sense to remove this indirection but I wanted to shorten the reference in a couple of places